dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/WebRumEventContextProvider.kt

```diff
@@ -14,6 +14,7 @@ import com.datadog.android.rum.internal.domain.RumContext
 internal class WebRumEventContextProvider {
 
     fun getRumContext(): RumContext? {
```

Comment:
Just thinking instead of dropping those events maybe we can still store them locally to be uploaded at a later time, that way events won't be lost

Reply:
Yes but the way the consumer works now is that if we do not have a valid rum context we are disabling the feature for good. Meaning that we will not allow other events anyway. This is mainly to avoid all the possible internal/dev logs as we discussed.

Reply:
Also there is no sense the way I see it to keep the events for later as we are only dropping those in case there is no RumContext. If there is no RumContext that means you forgot to enable the RUM feature or you are not doing this in the `Application#onCreate`. In the first case it will require a process restart anyway and in the latter I would say it is a initialization problem on user's end.